src/xargs/mod.rs

```diff
@@ -812,10 +812,17 @@ fn validate_positive_usize(s: &str) -> Result<usize, String> {
     }
 }
 
+#[allow(clippy::type_complexity)]
```

Comment:
can we avoid this please? thanks

Reply:
The alternatives that come to mind are:
1. Instead of returning a tuple from that function, return a new `Options` object.
2. Or pass in the `Options` object as a `mut` reference and modify it.

Would either of those be a good stylistic fit for the rest of the program?

Reply:
4 was probably already too much :)

Reply:
it can be removed now, no ?